### PR TITLE
update added in homepage

### PR DESCRIPTION
**Pull request by danieledebernardinDTD**
## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [ ] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Comment by danieledebernardinDTD** [comment]
@bfabio @denismarini quando potete, grazie

File: contents/home-page/home.yml [hunk 1]
```diff
@@ -131,42 +131,42 @@ opportunity:
       id: centri-facili
 heroCarouselNews:
   - id: 1
+    type: internal
+    category: Novità 
+    title: Fornitori Avvisi Cloud per Comuni e Scuole
+    description: " Il termine per l’attivazione dei contratti con i fornitori, sia per gli Avvisi Comuni che Scuole, è differito di tre mesi rispetto all’originario termine previsto negli Avvisi."
```
Comment (bfabio) [inline contents/home-page/home.yml:137]:
```suggestion
    description: "Il termine per l’attivazione dei contratti con i fornitori, sia per gli Avvisi Comuni che Scuole, è differito di tre mesi rispetto all’originario termine previsto negli Avvisi."
```

File: contents/home-page/home.yml [hunk 2]
```diff
@@ -131,42 +131,42 @@ opportunity:
       id: centri-facili
 heroCarouselNews:
   - id: 1
+    type: internal
+    category: Novità 
+    title: Fornitori Avvisi Cloud per Comuni e Scuole
+    description: "Il termine per l’attivazione dei contratti con i fornitori, sia per gli Avvisi Comuni che Scuole, è differito di tre mesi rispetto all’originario termine previsto negli Avvisi."
+    source: 'Decreto'
+    linkTo: 'https://areariservata.padigitale2026.gov.it/sfc/servlet.shepherd/document/download/0697Q0000065NQsQAM?operationContext=S1'
```
Comment (bfabio) [inline contents/home-page/home.yml:139]: È giusto sia un link diretto al download di un PDF?

Comment (danieledebernardinDTD) [reply contents/home-page/home.yml:139]: Si

File: contents/home-page/home.yml [hunk 3]
```diff
@@ -131,42 +131,42 @@ opportunity:
       id: centri-facili
 heroCarouselNews:
   - id: 1
+    type: internal
+    category: Novità 
+    title: Fornitori Avvisi Cloud per Comuni e Scuole
+    description: "Il termine per l’attivazione dei contratti con i fornitori, sia per gli Avvisi Comuni che Scuole, è differito di tre mesi rispetto all’originario termine previsto negli Avvisi."
+    source: 'Decreto'
```
Comment (bfabio) [inline contents/home-page/home.yml:138]:
```suggestion
    source: 'Decreto (Scarica PDF)'
```

Comment (danieledebernardinDTD) [reply contents/home-page/home.yml:138]: Ottimo

**Pull request merged**